### PR TITLE
Muon quickedit gets its initial values from context

### PR DESCRIPTION
**Pull request by AnthonyLim23**
**Description of work.**
When the quickedit is created, it will now get the initial values from the context. Making it easier to change the default settings in the future. 

**To test:**
Open frequency domain analysis
Load some data
Use the quick edit and make sure it behaves as expected
Do a transform 
Use the quick edit and make sure it behaves as expected

There is no associated issue.

In the release notes.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by robertapplin — CHANGES_REQUESTED** [review]
Looks good, can you add a test for this?

**Comment by robertapplin** [comment]
Tests are failing @AnthonyLim23

File: scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py [hunk 1]
```diff
@@ -52,6 +52,26 @@ def setUp(self):
     def tearDown(self):
         AnalysisDataService.Instance().clear()
 
+    def test_set_quickedit_from_context(self):
+        state = True
+        ylims = [0,10]
+        xlims = [5, 15]
+        self.context.set_error_all(state)
+        self.context.set_autoscale_all(state)
+        self.context.update_ylim_all(ylime)
```
Comment (AnthonyLim23) [inline scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py:61]:
```suggestion
        self.context.update_ylim_all(ylims)
```

File: scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py [hunk 2]
```diff
@@ -52,6 +52,26 @@ def setUp(self):
...
+        self.context.update_ylim_all(ylims)
+        self.context.update_xlim_all(xlims)
+
+        self.options.set_autoscale = mock.Mock()
+        self.options.set_errors = mock.Mock()
+        self.options.set_plot_x_range = mock.Mock()
+        self.options.set_plot_y_range = mock.Mock()
+
+        self.presenter.set_quickedit_from_context()
+        self.options.set_autoscale.assert_called_once_with(state)
```
Comment (AnthonyLim23) [inline scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py:70]:
```suggestion
        self.options.set_autoscale.assert_called_with(state)
```

File: scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py [hunk 3]
```diff
@@ -52,6 +52,26 @@ def setUp(self):
...
+        self.options.set_errors.assert_called_once_with(state)
```
Comment (AnthonyLim23) [inline scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py:71]:
```suggestion
        self.options.set_errors.assert_called_with(state)
```

File: scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py [hunk 4]
```diff
@@ -52,6 +52,26 @@ def setUp(self):
...
+        self.options.set_plot_x_range.assert_called_once_with(xlims)
```
Comment (AnthonyLim23) [inline scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py:72]:
```suggestion
        self.options.set_plot_x_range.assert_called_with(xlims)
```

File: scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py [hunk 5]
```diff
@@ -52,6 +52,26 @@ def setUp(self):
...
+        self.options.set_plot_y_range.assert_called_once_with(ylims)
```
Comment (AnthonyLim23) [inline scripts/test/Muon/plotting_canvas/plotting_canvas_presenter_test.py:73]:
```suggestion
        self.options.set_plot_y_range.assert_called_with(ylims)
```

**Review by robertapplin — APPROVED** [review]
Looks good to go

**Pull request merged**